tests/report_tests.py

```diff
@@ -230,3 +230,31 @@ def test_report_type_exclude(self):
         results = self._results(api.url_for(ReportsView, committee_id=committee_id, report_type=['-M3']))
         self.assertTrue(all(each['report_type'] in ['Q2', 'TER'] for each in results))
 
+    def test_ie_only(self):
+        factories.ReportsIEOnlyFactory(
+            committee_id='C90013723',
+            total_independent_contributions=200,
+            total_independent_expenditures=100,
+        )
+        db.session.flush()
+        number = 12345678902
+        factories.ReportsIEOnlyFactory(
```

Comment:
If we assign the result of this factory to a variable, we can use it later below:

```python
report = factories.ReportsIEOnlyFactory(
    beginning_image_number=12345678902,
    total_independent_contributions=200,
    total_independent_expenditures=100,
)
...
self.assertEqual(results[0]['total_independent_contributions'], report.total_independent_contributions)
```
That way, constants are only defined once.